crates/fluvio-version-manager/src/command/switch.rs

```diff
@@ -69,10 +69,18 @@ impl SwitchOpt {
 
         version_dir.set_active()?;
 
-        notify.done(format!(
-            "Now using Fluvio version {}",
-            version.to_string().bold(),
-        ));
+        if version.is_version_tag() {
+            notify.done(format!(
+                "Now using Fluvio version {}",
+                version.to_string().bold(),
+            ));
+        } else {
+            notify.done(format!(
+                "Now using Fluvio {} ({})",
+                version.to_string().bold(),
+                version_dir.manifest.version.to_string().bold(),
+            ));
+        }
```

Comment:
For version tags, say `fvm switch 0.10.11`, rendering the tag between parenthesis is redundant so its discriminated by the nature of the `Channel` instance provided.

We just want to render the specific tag for Channels.

```bash
fvm switch 0.10.15
...
done: Now using Fluvio version 0.10.15
```

And with Channels

```bash
fvm switch stable
...
done: Now using Fluvio stable (0.10.17)
```